packages/wonder-blocks-icon-button/docs.md

```diff
@@ -0,0 +1,581 @@
+                    pressed={true}
+                    disabled={false}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+            <td>
+                <IconButtonCore
+                    kind="tertiary"
+                    icon={searchIcon}
+                    alt="search"
+                    color={blue}
+                    light={false}
+                    hovered={false}
+                    focused={false}
+                    pressed={false}
+                    disabled={true}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+        </tr>
+        <tr style={{background: darkBlue, color: white, height: 60, verticalAlign: "middle"}}>
+            <th>Primary<br />(Light)</th>
+            <td>
+                <IconButtonCore
+                    kind="primary"
+                    icon={searchIcon}
+                    alt="search"
+                    color={blue}
+                    light={true}
+                    hovered={false}
+                    focused={false}
+                    pressed={false}
+                    disabled={false}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+            <td>
+                <IconButtonCore
+                    kind="primary"
+                    icon={searchIcon}
+                    alt="search"
+                    color={blue}
+                    light={true}
+                    hovered={true}
+                    focused={false}
+                    pressed={false}
+                    disabled={false}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+            <td>
+                <IconButtonCore
+                    kind="primary"
+                    icon={searchIcon}
+                    alt="search"
+                    color={blue}
+                    light={true}
+                    hovered={false}
+                    focused={false}
+                    pressed={true}
+                    disabled={false}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+            <td>
+                <IconButtonCore
+                    kind="primary"
+                    icon={searchIcon}
+                    alt="search"
+                    color={blue}
+                    light={true}
+                    hovered={false}
+                    focused={false}
+                    pressed={false}
+                    disabled={true}
+                >
+                    Label
+                </IconButtonCore>
+            </td>
+        </tr>
+    </tbody>
+</table>
+```
+
+IconButtonCores, Red:
+```js
+const Color = require("wonder-blocks-color").default;
+const IconButtonCore = require("./components/icon-button-core.js").default;
+const searchIcon = `M7.73732912,6.67985439 C7.75204857,6.69246326 7.76639529,
```

Comment:
Maybe extract this out into a helper file somewhere.

Reply:
I don't think we need to worry about that for now - it's all going to get re-written once the Icon implementation comes in!